graftm/bootstrapper.py

```diff
@@ -86,22 +102,47 @@ def generate_hmm_from_contigs(self, contig_files, output_hmm_file):
             # Now have a fasta file of ORFs.
             # Check to make sure the file is not zero-length
             orfs.flush()
-            if os.stat(orfs.name).st_size == 0:
-                logging.warn("Failed to find any matching ORFs in the bootstrap contigs")
+            
+            if len(seqio.read_fasta_file(orfs.name)) <= 1:# Just to build on this, you need to check if there is > 1 hit
```

Comment:
only matters when method is hmmsearch, ok to bootstrap if diamond is being used yeh?